src/Facemesh/index.js

```diff
@@ -64,7 +64,10 @@ class Facemesh extends EventEmitter {
     const { flipHorizontal } = this.config;
     const predictions = await this.model.estimateFaces(input, flipHorizontal);
     const result = predictions;
+    // Soon, we will remove the 'predict' event and prefer the 'face' event. During
+    // the interim period, we will both events.
     this.emit("predict", result);
+    this.emit("face", result);
```

Comment:
Nonblocking comment:
+ I could also entertain the idea doing a kind of "friendly deprecation message" where if people are listening for the 'predict' event, that we could return `null` or `[]` and log "predict is no longer supported: please use .on(`face`, ...)"?